restapi/configure_console.go

```diff
@@ -128,6 +128,8 @@ func configureAPI(api *operations.ConsoleAPI) http.Handler {
 	registerAdminTiersHandlers(api)
 	//Register Inspect Handler
 	registerInspectHandler(api)
+	// Register nodes handlers
+	registersNodesHandler(api)
```

Comment:
Very minor issue here, should we change the function name to `registerNodesHandler` to keep it consistent with the other functions?

```suggestion
	registerNodesHandler(api)
```